examples/drupal_loadtest.rs

```diff
@@ -97,12 +97,12 @@ async fn drupal_loadtest_front_page(client: &mut GooseClient) {
             }
             Err(e) => {
                 eprintln!("failed to parse front page: {}", e);
-                client.set_failure(&GooseMethod::GET, "/");
+                client.set_failure(&response.request);
```

Comment:
I am not yet understanding why we cannot get a shared reference for the express purpose of only sending this struct member. That should be safe.

Reply:
The earlier call to `t.text().await` takes partial ownership of the request.request object. If we try and pass the entire GooseRequest object to `set_failure(&request)` we get the following compilation error:

```
move occurs because value has type `reqwest::async_impl::response::Response`, which does not implement the `Copy` trait
```

I believe this is because internally it's calling `to_bytes`, and it works similar to `into_bytes` in Rust's std String:
https://doc.rust-lang.org/std/string/struct.String.html#method.into_bytes

> This consumes the String, so we do not need to copy its contents.